src/utils/frontend-api-client/work-orders.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+
+export const getWorkOrders = async () => {
+  const { data } = await axios.get('/api/work-orders')
```

Comment:
I think the idea is to match these endpoints with the service api endpoint. So if you look on swagger i think this exists now as `/repairs`

Reply:
Yes having the paths on our side match with the service API will make it easier to have a catch-all API route on our Node server. (It just "forwards" the path, params, method, body etc. on to the service API, and doesn't need to maintain any path mappings or things like that.)